cmd/cli/kubectl-kyverno/processor/result.go

```diff
@@ -1,6 +1,8 @@
 package processor
 
 import (
+	"fmt"
```

Comment:
do we need to keep changes in this file ?

Reply:
The printing logic from this file is being moved to `command.go`. `fmt` will be removed here